### ISSUE TITLE
New install first repo clone, get error

### ISSUE THREAD

**Issue by spasblitz** [gitextensions/gitextensions#7839]
* New company laptop
* installed 64bit git bash (2.25.1) successfully
* Installed Git Extensions 3.3.1.7897 successfully
* Choose Clone Repository from application, or through right click menu in file explorer, get error.  
* When closing error dialog, clone repo dialog box does successfully show

## Expected behaviour
I expected to not get error box :)

## Error Details
```
System.ArgumentException: Illegal characters in path.
   at System.IO.Path.CheckInvalidPathChars(String path, Boolean checkAdditional)
   at System.IO.Path.Combine(String path1, String path2)
   at GitUI.CommandsDialogs.FormClone.ToTextUpdate(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnTextChanged(EventArgs e)
   at System.Windows.Forms.Control.set_Text(String value)
   at System.Windows.Forms.TextBoxBase.set_Text(String value)
   at System.Windows.Forms.TextBox.set_Text(String value)
   at GitUI.CommandsDialogs.FormClone.FromTextUpdate(Object sender, EventArgs e)
   at GitUI.CommandsDialogs.FormClone.OnRuntimeLoad(EventArgs e)
   at GitUI.GitExtensionsForm.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at ResourceManager.GitExtensionsFormBase.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Environment
- Git Extensions 3.3.1.7897
- Build 5a97671645532bcedc443bac7b727f40db47cb5c
- Git 2.25.1.windows.1
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4018.0
- DPI 144dpi (150% scaling)

**Comment by pmiossec** [comment]
Duplicate of #7528. Somewhat fixed in master with #7633

**Comment by gerhardol** [comment]
What clone path?

**Comment by spasblitz** [comment]
Gerhard,

Through a bit of testing the last 10 minutes, the problem only happens when my clipboard contains my ssh key…..

This is the key contents….

ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDVN9a+Tg6zcqEhuALxrqtuf23OGVK0cmQyGpjiy93GMXTZANPLZun2b5VfEZ0e3rkBLYj9KXe2mYHmn9OUgjXjDaWyAtAQAzvTpROpwe/VTYAZQOci3gbtBDobM+8E52aEpNy1IIZHN2EUTQJmhcvSxJe0injFJOeKNJq96OuIZeprOM0uagCHkHAu8QzjEI4B6XUIFIyJFFHcGrqJ8Fld6iIxbf3UEAxNQ/qhazOtQfKbhDkoWnHG3j6TKXEBFhQn+nsMpTG/Hs//ykuGFYNk2ttMfOFJqRLqcY0pyoIQEHen4D9WoGGbnwXphvP/skyYyziQRqZxWYmjXy3W2fHN1wyCGO3P3FIip3jl4RBDk2kzXg3tAiAwuk2Tsg86j7WzjC4dvi48MgYXhytp/jMxJKJQQoPTQUu04os/VP54II81JWyRFOB57aeNtRys57wWWlWTZz1UVzCEAnImODZPMnTb24Lu7M3P7tlNC2GKHGExazIDnsWG9q9RjkwMnpW1xYN62Nr8TsclE0k4bwy37RKKzBIMFWey82a1HHjieKufSY+q0VfZHI6uamtwLS0kh5kNuEDtdEj7rr3Ey+2eyZZTKKlcM3jfpyCABlZJJED377aRiZNgPhiuQGkufga4emmaBJ7YoLtWGcUZpxbF5qG5zneDqpzHd65QkN1N2w== bill.schmitt@infor.com<mailto:bill.schmitt@infor.com>

when clipboard is clear, clicking clone repository gives no error at all.  I was getting the error before the clone dialog box was showing.  Without that key in my clipboard, I receive no error.

Thanks,

**Comment by gerhardol** [comment]
(removed the ssh path from your post)

The ssh contains chars not allowed in paths so that cannot be allowed. But did you press clone directly without changing the url/path?
Curious as there has been several occurrences lately, is there anything that can be improved?

**Comment by mstv** [comment]
I guess, we should simply ignore the clipboard if it does not contain a valid path.

**Comment by spasblitz** [comment]
So, my clipboard contained invalid chars, and I clicked clone repository button and received error about an invalid character....BEFORE I was ever given an opportunity to enter a clone path.  AFTER I click the error away, the clone box would appear successfully.   I never entered a path, I came here :)

**Comment by mstv** [comment]
I cannot reproduce this issue neither with a current master nor with 3.1.1.6049.
Might have been resolved by #7633.
Exceptions accessing the clipboard have already been catched.
But the check `CanBeGitURL` was incomplete -- improved in #7876.
